cmd/containerd-shim-runhcs-v1/options/runhcs.proto

```diff
@@ -61,3 +61,30 @@ message ProcessDetails {
 	uint64 user_time_100_ns = 8;
 	string exec_id = 9;
 }
+
+message Statistics {
```

Comment:
I agree with @Random-Liu we should put this in the "stats" package. If you look at my updated PR I moved it. Do you want to just do that now? Its a very minor change

Reply:
That's fine, I can do it here.